src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,35 @@
+.stars {
+  height: 16px;
+  display: flex;
+  flex-direction: row;
+
+  &__star-active {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star-active.svg);
+    box-sizing: border-box;
+    align-items: center;
+    margin: 0 4px 0 0;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    box-sizing: border-box;
+    align-items: center;
```

Comment:
Similarly to the '.stars__star-active' class, the 'align-items' property should be removed from '.stars__star' for the same reason.